gooey-gui/app/components/bulkProgress/BulkProgressCard.css

```
@@ -0,0 +1,510 @@
+.bulk-progress-card {
+  --bulk-progress-detail-label-width: 7.5rem;
+  background: #fff;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)